client/server/render/index.js

```diff
@@ -122,24 +122,13 @@ export function render( element, key = JSON.stringify( element ), req ) {
 
 		return renderedLayout;
 	} catch ( ex ) {
-		if ( process.env.NODE_ENV === 'development' ) {
-			throw ex;
-		} else {
-			try {
-				req.context.store.dispatch(
-					logToLogstash( {
-						feature: 'calypso_ssr',
-						message: 'Exception thrown on render',
-						user_id: req.context.user?.ID,
-						extra: {
-							message: ex.message,
-							stack: ex.stack,
-						},
-					} )
-				);
-			} catch {
-				// Failed to log the error, swallow it so it doesn't break anything. This will serve
-				// a blank page and the client will render on top of it.
+		try {
+			req.logger.error( ex );
```

Comment:
By using the logger, are we preserving all the fields that were present in the `logToLogstash()` action above?

Reply:
Kind of:
- `feature` will go away, but I don't think it provided anything useful
- `message` will go away, also not super useful IMO.
- `user_id` will also go away, but given we don't do SSR render if `context.user` is present it was always an empty field (looking at logs from the last 90 days confirms this)
- `extra` will also go away, but we'll have `err` with the error info.